.gitignore

```diff
@@ -5,10 +5,14 @@ coverage
 /dist
 /test/regressions/screenshots/output
 /test/selenium-output
+.eslintcache
 
 # OS files
 .DS_STORE
 
 # npm files
 node_modules
 *.log
+
+# Editor files
+.idea
```

Comment:
Oh, never mind. I saw it in master there, sou mindlessly coppied. If you did decision not to include it in project gitignore, absolutely no problem for me too!